### PR TITLE
Tbg ticket3

### PR DESCRIPTION
**Pull request by emmyzzang**

### CODE REVIEW

File: myTest.txt [hunk 1]
```diff
@@ -1,2 +1,6 @@
 
-yooo
\ No newline at end of file
+yooo
+YETI WAS HERE
+myBiggz
+while loop
+hahahah requirement goes here. UNIQUE.
```
Comment (eemike07) [inline myTest.txt]: Unneccessarily hahahahaah....please remove

File: myTest.txt [hunk 2]
```diff
@@ -1,2 +1,6 @@
 
-yooo
\ No newline at end of file
+yooo
+YETI WAS HERE
+myBiggz
+while loop
```
Comment (eemike07) [inline myTest.txt]: remove while loop

File: myTest.txt (hunk 1 above)
Comment (emmyzzang) [reply myTest.txt]: will do!

File: myTest.txt (hunk 2 above)
Comment (emmyzzang) [reply myTest.txt]: will do

**Pull request merged**